src/interner.rs

```diff
@@ -254,6 +271,7 @@ impl<T: ?Sized> Interner<T> {
 
 /// Constructors to control the backing `HashSet`'s hash function
 impl<T: Eq + Hash + ?Sized, H: BuildHasher> Interner<T, H> {
```

Comment:
I just noticed that I forgot to remove the bounds here (and more problematically, on the `_raw` access as well). Unrelated to this PR, just note to self that I should fix this before publishing this update.